src/blocks/focused_window.rs

```diff
@@ -213,7 +221,11 @@ impl Block for FocusedWindow {
                 }
             }
         };
-        self.text.set_text(escape_pango_text(out_str));
+        let values = map!(
+            "composed" => Value::from_string(escape_pango_text(out_str))
```

Comment:
Should we also add "title" and "marks"?

Reply:
I'm not convinced they're useful (since "composed" comes from some logic that makes sense) but they don't hurt so added.

Reply:
Yes the existence of show_marks is probably why no one tried to add a format string previously, since it's a bit tricky to make it fit in with the furthest string.

It makes as much sense as the music block having "artist", "title" and "combo". Also it will allow showing both marks and the title, which isn't possible right now (although not sure if anyone would do that)

Reply:
> Also it will allow showing both marks and the title, which isn't possible right now (although not sure if anyone would do that)

Well, done. Anything else? 
Don't love having too many pending PRs, my fork is deviating fast :-)